<ui version="4.0" >
 <class>QtAssistantDock</class>
 <widget class="pDockWidget" name="QtAssistantDock" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/QtAssistant.qrc" >
    <normaloff>:/icons/assistant.png</normaloff>:/icons/assistant.png</iconset>
  </property>
  <property name="windowTitle" >
   <string>Qt Assistant</string>
  </property>
  <widget class="QWidget" name="wCentral" >
   <layout class="QVBoxLayout" name="vlCentral" >
    <property name="spacing" >
     <number>6</number>
    </property>
    <property name="margin" >
     <number>9</number>
    </property>
    <item>
     <widget class="QComboBox" name="cbFilter" />
    </item>
    <item>
     <widget class="QTabWidget" name="twPages" >
      <property name="tabPosition" >
       <enum>QTabWidget::South</enum>
      </property>
      <widget class="QWidget" name="wContents" >
       <attribute name="title" >
        <string>&amp;Contents</string>
       </attribute>
       <layout class="QVBoxLayout" name="vlContents" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>9</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="wIndex" >
       <attribute name="title" >
        <string>&amp;Index</string>
       </attribute>
       <layout class="QVBoxLayout" name="vlIndex" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>9</number>
        </property>
        <item>
         <widget class="QLabel" name="lLookFor" >
          <property name="text" >
           <string>&amp;Look for:</string>
          </property>
          <property name="buddy" >
           <cstring>leLookFor</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leLookFor" />
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="wBookmarks" >
       <attribute name="title" >
        <string>&amp;Bookmarks</string>
       </attribute>
       <layout class="QVBoxLayout" name="vlBookmarks" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>9</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="pbProgress" />
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pDockWidget</class>
   <extends>QDockWidget</extends>
   <header>fresh.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/QtAssistant.qrc" />
 </resources>
 <connections/>
</ui>
